hack/test-e2e.sh

```diff
@@ -5,7 +5,13 @@ if [ -n "${IMAGE_CLUSTER_LOGGING_OPERATOR:-}" ] ; then
   source "$(dirname $0)/common"
 fi
 
+IMAGE_STREAM_CLUSTER_LOGGING_OPERATOR=$(oc get is pipeline -n $OPENSHIFT_BUILD_NAMESPACE -o jsonpath='{.status.dockerImageRepository}')
```

Comment:
The required image should be available with this var [1] as I disussed with @bbguimaraes.  If this is not working as advertised we should talk it over with the him, @stevekuznetsov and the rest of the test platform team

[1] https://github.com/openshift/ci-operator/blob/master/TEMPLATES.md#image_component

Reply:
Would we want the `IMAGE_` or `LOCAL_IMAGE_` in this case though? which is the one that was built as part of this CI run?

Reply:
@jcantril: On second thought, that variable is available for the
(`ci-operator`) template, but it won't be visible to the test unless it is
listed as a parameter (it is not).

The `IMAGE_FORMAT` variable [is][0] available in the test, so it could be used.
The [format][1] is very similar to `IMAGE_<component>`.

I don't know if this is how we want to deploy and test an operator, however.
@smarterclayton may.

[0]: https://github.com/openshift/release/blob/d71c33b7b096c33da6d2aa8948160e9169242908/ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml#L102-L103
[1]: https://github.com/openshift/ci-operator/blob/master/TEMPLATES.md#image_format

Reply:
You are allowed to use IMAGE_FORMAT in your own testing, because you may want to access tags or images that aren't part of the release payload.  For instance, if you build your own tests image, or want to access official components directly like the CLI image or the tests image

Reply:
Generally use `IMAGE_X` or `LOCAL_IMAGE_X` over `IMAGE_FORMAT`, but we won't change the templates to add every possible one, so `IMAGE_FORMAT` is the tool we give the operator tests.